<launch>
    <arg name="user" default="$(env USER)" />
    <arg name="pcd_file" default="/home/$(arg user)/catkin_ws/pcd/scans_roi_plat.pcd" />
    <arg name="output_pcd_file_filtered" default="/home/$(arg user)/catkin_ws/pcd/map_filtered.pcd" />
    <arg name="output_pcd_file_top" default="/home/$(arg user)/catkin_ws/pcd/map_top.pcd" />
    <arg name="output_pcd_file_down" default="/home/$(arg user)/catkin_ws/pcd/map_down.pcd" />
    <arg name="z_threshold" default="1.0" />

    <param name="pcd_file" value="$(arg pcd_file)" />
    <param name="z_threshold" value="$(arg z_threshold)" />

    <node pkg="pcd_divider" name="pcd_divider_node" type="pcd_divider_node" output="screen">
        <param name="pcd_file" value="$(arg pcd_file)" />
        <param name="z_threshold" value="$(arg z_threshold)" />
    </node>
</launch>
